pollen_data_gen/pollen_data_gen/simple.py

```diff
@@ -1,11 +1,97 @@
+# import sys
 import json
-import dataclasses
-from typing import Dict, Union, Optional, Any
+from typing import Dict, Union, Optional, Any, List, Sequence, TextIO
 from json import JSONEncoder
 from mygfa import mygfa
+from . import depth
 
 
-SimpleType = Optional[Dict[str, Union[bool, str, int]]]
+SimpleType = Optional[
+    Union[str, Dict[str, Sequence[object]], List[int], List[Union[int, str]]]
+]
```

Comment:
Perhaps this could use a quick comment about what it's for? I think it may be the general type for all JSON-serializable objects produced here, for instance.